### ISSUE TITLE
Flexible Content Field labels switch unexpectedly

### ISSUE THREAD

**Issue by felixdorner** [elliotcondon/acf#577]
Hey Elliot, 

my field architecture consist of multiple nested flexible content fields. In the backend the label for the flexible content fields show a strange behavior as they switch and show the label of the last subfield within the last nested flexible content field.

I testet it on a blank WordPress installation. There are no Javascript errors but there are some PHP warnings when I look into the debug.log:

```
[10-Oct-2016 14:30:30 UTC] PHP Warning:  array_pop() expects parameter 1 to be array, string given in /wordpress/wp-content/plugins/advanced-custom-fields-pro/pro/fields/flexible-content.php on line 1347
[10-Oct-2016 14:30:30 UTC] PHP Stack trace:
[10-Oct-2016 14:30:30 UTC] PHP   1. {main}() /wordpress/wp-admin/admin-ajax.php:0
[10-Oct-2016 14:30:30 UTC] PHP   2. do_action() /wordpress/wp-admin/admin-ajax.php:91
[10-Oct-2016 14:30:30 UTC] PHP   3. call_user_func_array:{/wordpress/wp-includes/plugin.php:524}() /wordpress/wp-includes/plugin.php:524
[10-Oct-2016 14:30:30 UTC] PHP   4. acf_field_flexible_content->ajax_layout_title() /wordpress/wp-includes/plugin.php:524
[10-Oct-2016 14:30:30 UTC] PHP   5. array_pop() /wordpress/wp-content/plugins/advanced-custom-fields-pro/pro/fields/flexible-content.php:1347
```

I am using ACF Pro Version 5.4.6.

Here is my field group export as JSON:

[acf-export-2016-10-10.json.zip](https://github.com/elliotcondon/acf/files/519725/acf-export-2016-10-10.json.zip)

Here is a screen capture video of the issue:

[acf-flexible-content-labels-flickering.mp4.zip](https://github.com/elliotcondon/acf/files/519899/acf-flexible-content-labels-flickering.mp4.zip)

**Comment by elliotcondon** [comment]
Hi @felixdorner 

Thanks for the bug report. I believe this is the same issue as one I have recently fixed.

If possible, can you please download and test the latest build of ACF PRO?
1. Login to your account online: http://www.advancedcustomfields.com/my-account
2. Click on the 'see all versions' link next to your license
3. Download the latest 5.4.7-RC1 version​​

**Comment by felixdorner** [comment]
Hi @elliotcondon,

thanks for your help. Version 5.4.7-RC1 fixed the issue.

Cheers,
Felix

**Comment by elliotcondon** [comment]
Awesome!
